README.md

```diff
@@ -1,4 +1,44 @@
-﻿# Открытое письмо специалистов IT-индустрии в защиту фигурантов «московского дела»
+# Public letter from IT specialists to the defense of the people involed in "Mocsow criminal case".
+
+[27 July 2019](https://ovdinfo.org/news/2019/07/27/miting-u-merii-moskvy-27-iyulya-2019-goda-i-ego-posledstviya-onlayn) people went out peacefully without weapons, just to express their disagreement with [prohibition of independent candidates](https://www.bbc.com/russian/features-49127945) before the elections to the Moscow City Duma. This event was an occasion to initiate a new round of political repressions and to open a case of "mass riots", 13 people were involved in.
+
+Journalists, witnesses and participants note that the action was performed exclusively peacefully: there were no any damage to the property or any violence from the demonstrators. But this didn't prevent interrogators from bringing charges, and judges, rejecting all requests by the defense, to convict - in incredibly short time. [Konstantin Kotov (4 years in prison)] (https://www.interfax.ru/russia/675323), [Ivan Podkopaev (3 years in prison)] (https: // www. interfax.ru/moscow/674950), [Pavel Ustinov (3.5 years in prison)] (https://www.interfax.ru/russia/676593).
+
+On September 17 and 18, 2019, priests and teachers were indignant by the absurdity of this and they issue public letters in which they express their opinion: the case must be stopped, and the innocent must be released. We, IT-specialists, see a request for all professional communities to express their position on this matter, and we also cannot remain silent.
+
+One of the defendants in the “Moscow affair” is our colleague, Aidar Gubaidulin. Aidar graduated from Moscow Institute of Physics and Technology and works at Sberbank-Technology. At the same time, our other colleague, programmer Konstantin Kotov, was convicted: he received 4 years in prison for ** participation ** in four actions, because, according to the prosecution, he pursued "the goal of creating a real threat to the constitutionally protected rights and freedoms of man and citizen" .
```

Comment:
Fix markdown: `** participation **` -> `**participation**`

```suggestion
One of the defendants in the “Moscow affair” is our colleague, Aidar Gubaidulin. Aidar graduated from Moscow Institute of Physics and Technology and works at Sberbank-Technology. At the same time, our other colleague, programmer Konstantin Kotov, was convicted: he received 4 years in prison for **participation** in four actions, because, according to the prosecution, he pursued "the goal of creating a real threat to the constitutionally protected rights and freedoms of man and citizen" .
```